modules/get-started/pages/release-notes/redpanda.adoc

```diff
@@ -7,68 +7,6 @@ This topic includes new content added in version {page-component-version}. For a
 * xref:redpanda-cloud:get-started:whats-new-cloud.adoc[]
 * xref:redpanda-cloud:get-started:cloud-overview.adoc#redpanda-cloud-vs-self-managed-feature-compatibility[Redpanda Cloud vs Self-Managed feature compatibility]
 
-== Iceberg topics with AWS Glue
+== Client connection monitoring
```

Comment:
I would call it 'Connected client monitoring' as we have in Aha etc, because the real use case is looking at things 'per client' not 'per connection' for currently connected clients.  You're trying to find the bad clients. Technically a connection is a connection to each broker from a client.